dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/bnd.bnd

```diff
@@ -42,3 +42,11 @@ javac.target: 1.8
   com.ibm.wsspi.org.osgi.service.component,\
   com.ibm.wsspi.org.osgi.service.component.annotations,\
   javax.activation:activation;version=1.1
+
+-testpath: \
+	org.hamcrest:hamcrest-all;version=1.3, \
+	org.jmock:jmock-junit4;strategy=exact;version=2.5.1, \
+	../build.sharedResources/lib/junit/old/junit.jar;version=file, \
+	com.ibm.ws.junit.extensions;version=latest, \
+	com.ibm.websphere.org.eclipse.microprofile.rest.client.1.1;version=latest
```

Comment:
This looks like something that log left behind.  I don't think MP rest client should be a dependency.